### PR TITLE
[REVIEW] Auto-add new Issues and PRs to cuspatial's project

### PR DESCRIPTION
**Pull request by jarmak-nv**
The new project boards should have every issue/PR within them. This workflow makes the addition of the issues/PRs automatic rather than the current manual process.

### CODE REVIEW

**Review by thomcom — APPROVED** [review]
Nice!

**Review by harrism — APPROVED** [review]
Neat!

**Comment by harrism** [comment]
```
Error: Request failed due to following response errors:
 - Resource not accessible by integration
```

**Comment by jarmak-nv** [comment]
For viz: we need to create a new org secret that has the required access, the workflow fails using the existing secret so until we get that settled this will sit.

**Comment by harrism** [comment]
@jarmak-nv can you retarget this to 22.10? We missed the boat for 22.08.

**Comment by jarmak-nv** [comment]
@harrism yeah I'll update the target once the new key is created and I can update that at the same time. This will probably be sitting until after 22.08 gets fully released.

**Comment by thomcom** [comment]
I went ahead and changed the base.

**Comment by harrism** [comment]
@jarmak-nv you will need to merge 22.10 into your branch and push in order to complete the move to 22.10.

**Comment by harrism** [comment]
@jarmak-nv what's happening with this?

File: .github/workflows/add_issue_to_project.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+jobs:
+  add-to-project:
+    name: Add issue or PR to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.3.0
+        with:
+          project-url: https://github.com/orgs/rapidsai/projects/41
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (raydouglass) [inline .github/workflows/add_issue_to_project.yml:20]: I've added a secret called `ADD_TO_PROJECT_GITHUB_TOKEN` for this purpose. Kindly update the name in the workflow and you should be good to go.

Comment (jarmak-nv) [reply .github/workflows/add_issue_to_project.yml]: Done ty!

**Review by harrism — APPROVED** [review]

**Review by thomcom — APPROVED** [review]

**Comment by harrism** [comment]
@gpucibot merge

**Review by msadang — APPROVED** [review]
Lgtm

**Pull request merged**